### DISCUSSION TITLE
Merge queue: `git log --pretty="%B"`

### DISCUSSION THREAD

**Discussion by botondbutuza** [community/community#48167] [Pull Requests]
### Select Topic Area

Bug

### Body

Hello!

When running `git log --pretty="%B" origin/master...HEAD --merges`, there's a difference in output between PRs that have been merged with merge queue, and ones that have been merged "conventionally". It seems like there's an extra new line added to the subject of the commit. This can also be seen with `--pretty="%b"`, where merge queue merges have a blank line between outputs, whereas non-merge queue merges don't.

We have some scripts that rely on the non-merge queue output, which are now broken. :) From my understanding of git conventions, there should be an extra blank line, but for backwards compatibility, maybe this should be fixed to work as it does now.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @botondbutuza ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
